api/rpc/stat/stat.go

```diff
@@ -29,9 +29,16 @@ func New(cfg Config) (*Stat, error) {
 	return &Stat{conn: c}, err
 }
 
-// ExecuteQuery implements business logic for StateServer interface
+// ExecuteQuery implements business logic for StatServer interface
```

Comment:
Sorry @chrisccoy -- totally my fault here. I was thinking about the influx package, not the stat package when I wrote that.